### PR TITLE
Reboot to rescue endpoint

### LINKED ISSUE #1030: Implement rescue endpoint

### PR DESCRIPTION
**Pull request by abemassry**
closes #1030 
image

### CODE REVIEW

File: src/linodes/linode/layouts/RescuePage.js [hunk 1]
```diff
@@ -103,7 +118,7 @@ export class RescuePage extends Component {
           /dev/{AVAILABLE_DISK_SLOTS[index]}
         </label>
         <div className="col-sm-10">
-          {disks[device].label}
+          <span id={disks[device].id}>{disks[device].label}</span>
```
Comment (eatonphil) [inline src/linodes/linode/layouts/RescuePage.js]: Why are you setting id here?

Comment (abemassry) [inline src/linodes/linode/layouts/RescuePage.js]: I refactored that twice and forgot to remove it, good catch

File: src/linodes/linode/layouts/RescuePage.js [hunk 2]
```diff
@@ -91,6 +92,20 @@ export class RescuePage extends Component {
     }
   }
 
+  async rebootToRescue() {
+    const { diskSlots } = this.state;
+    const { dispatch } = this.props;
+    const linode = this.getLinode();
+    const diskArray = {};
```
Comment (mparke) [inline src/linodes/linode/layouts/RescuePage.js]: can we name this something other than `diskArray`, since it's an object

Comment (briansteffens) [inline src/linodes/linode/layouts/RescuePage.js]: Better yet, diskLinkedList. That'll really throw people off our trail.

Comment (eatonphil) [inline src/linodes/linode/layouts/RescuePage.js]: Like `disks` so you can just put it into the dict later that expects a `disks` key

Comment (eatonphil) [inline src/linodes/linode/layouts/RescuePage.js]: @briansteffens I'd merge it if it were diskStack

Comment (briansteffens) [inline src/linodes/linode/layouts/RescuePage.js]: @peaton isn't it more of a counter?

Comment (eatonphil) [inline src/linodes/linode/layouts/RescuePage.js]: No. It's a very different thing. A counter is not idiomatic Javascript.

**Pull request merged**